src/main/webapp/app/overview/course-conversations/dialogs/channels-create-dialog/channel-form/channel-form.component.html

```diff
@@ -65,6 +65,31 @@
                         ></small>
                     </div>
                 </div>
+                <!-- Course-Wide Channel -->
```

Comment:
this option should only be shown for moderators of the course (tutors, instructors, admins). All others should not even see the option